packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -899,12 +899,22 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     goBallistic(0.0);
   }
 
-  void pointerScroll(double delta) {
+  void pointerScroll(double delta, bool animatePointerScroll) {
     // If an update is made to pointer scrolling here, consider if the same
     // (or similar) change should be made in
     // ScrollPositionWithSingleContext.pointerScroll.
     assert(delta != 0.0);
 
+    if (animatePointerScroll) {
+      double currentPosition = _outerPosition?.pixels ?? 0.0;
```

Comment:
Another point for https://github.com/flutter/flutter/issues/102003, this kind of info should be more accessible.